.travis.yml

```diff
@@ -45,3 +45,12 @@ after_success:
 notifications:
   email: false
 
+jobs:
+  include:
+    - stage: Code Quality
+      php: 7.2
+      env: STATIC_ANALYSIS
+      install: travis_retry composer update --prefer-dist --prefer-stable
+      before_script:
+        - travis_retry composer require --dev --prefer-dist --prefer-stable phpstan/phpstan:^0.9
```

Comment:
I suggest using a specific phpstan/phpstan version here (`0.9.1` at the time of writing). If new PHPStan version can detect new types of issues, it would break the build for unrelated PR.

Reply:
I disagree - by locking to a specific version, you'd deprive yourself of bugfixes and improvements of new versions. My policy for 0.x.y versions is the following:

Bumping number y can bring new tool features (like a new error formatter or a new CLI option) and bugfixes/improvements that lead to better understanding of the code.

Bumping number x usually brings new rules that detect new types of defects in the code and can also introduce BC breaks.

So here I'd require `^0.9`- Although almost any change can "break the build", it's usually welcome by the users that the code is now understood better and that maybe some type of error can no longer need to be ignored etc.